script/Arcueid/Day 8/08_00_ARC08_3_2.txt

```diff
@@ -252,7 +252,8 @@ There's still ten minutes left until lunch break.
 [sha:0bc7c06018d8ea9c4ceb08210bbd424534d378c6]{
 -- Page 211, Offset 20194.
 -- 逃げたな、このーーー！」
-He ran off! That little!"
+// Not 100% literal but please the この I shan't
+You think you can run, huh?!"
```

Comment:
The above line could probably be a bit spruced up

Reply:
Anything in mind? I did consider 
> Hey! Where'd Shiki go?!

or something to that effect

Reply:
Something like this, I guess
>Waaait, where'd Shiki run off to?!